app/views/volunteers/_form.html.slim

```diff
@@ -87,16 +87,13 @@ fieldset
 fieldset
   legend= t_attr(:group_accompaniment, Volunteer)
   .row
-    - Volunteer::GROUP_ACCOMPANIMENTS.each do |group_accompaniment|
-      .col-xs-6.col-md-3
-        = f.input group_accompaniment
+    .col-xs-12.
```

Comment:
nit picking: there is an extra dot after 12

Reply:
done